### PR TITLE
Revise documentation on running Pelican servers in docker container

### LINKED ISSUE #1076: Update docs to run Pelican servers via docker container
* For running origins, add `-f` command or include more tips on what flags to include
* For all container commands, fix the typo where `serve` command was missing
* Once #1075 is merged in, confirm that the latest build works
* Note in the doc that one can run the container with the specific Pelican version

### PR DESCRIPTION
**Pull request by haoming29**
Closes #1076

### CODE REVIEW

File: docs/pages/install/docker.mdx [hunk 1]
```diff
@@ -82,3 +89,52 @@ CONTAINER ID   IMAGE  COMMAND   CREATED  STATUS   PORTS    NAMES
...
 docker stop pelican-director
 ```
+
+## Configure Pelican server in a container
+
+There are two ways to configure a Pelican server running in a container. One is through the environment variables, the other is by passing a `pelican.yaml` configuration file to the container. This section includes instructions for both.
+
+### Configure via environment variables
+
+All Pelican configuration parameters described in [parameters page](../parameters.mdx) can be passed to a Pelican server as environment variables. For a configuration parameter, the corresponding environment variable is `PELICAN_<PARAMETER_NAME>`, where `PELICAN` is the prefix, and `<PARAMETER_NAME>` is the name of the parameter, with dots `.` replaced by underscores `_`. For example, a Pelican configuration named `Logging.Level` has a corresponding environment variable named `PELICAN_LOGGING_LEVEL`.
```
Comment (jhiemstrawisc) [inline docs/pages/install/docker.mdx:99]: This actually isn't quite true, especially for lists of objects like our `Origin.Exports` block. Maybe just say "Most Pelican configuration parameters... "

Comment (haoming29) [reply docs/pages/install/docker.mdx:99]: Sounds good. I'll leave a note about the `object` type params

File: docs/pages/install/docker.mdx [hunk 2]
```diff
@@ -82,3 +89,54 @@ CONTAINER ID   IMAGE  COMMAND   CREATED  STATUS   PORTS    NAMES
...
+
+> For admins that use Pelican images built for OSDF (with `osdf` prefix to the image tag, such as `osdf-origin`). The environment variables are prefixed by `OSDF` instead of `PELICAN`. The example above should then be `OSDF_LOGGING_LEVEL`.
+
+To pass environment variables to the Docker container, append `-e` flag to your `docker run` command. For example, if you want to pass  `PELICAN_LOGGING_LEVEL` to the container, run:
+
+```bash
+docker run -it -e PELICAN_LOGGING_LEVEL=debug -p 8444:8444 -p 8443:8443 -v /path/to/your/data/:/tmp/pelican --name=pelican-origin hub.opensciencegrid.org/pelican_platform/origin:latest serve -v /tmp/pelican:/foo/bar -f <federation>
+```
+
+There are another ways to pass environment variables to a Docker container, see details in [Docker documentation](https://docs.docker.com/reference/cli/docker/container/run/#env).
```
Comment (turetske) [inline docs/pages/install/docker.mdx:109]:
```suggestion
There are other ways to pass environment variables to a Docker container, see details in [Docker documentation](https://docs.docker.com/reference/cli/docker/container/run/#env).
```

File: docs/pages/install/docker.mdx [hunk 3]
```diff
@@ -82,3 +89,54 @@ CONTAINER ID   IMAGE  COMMAND   CREATED  STATUS   PORTS    NAMES
...
+
+> Some of Pelican configuration parameters with the `object` type can not be passed as environment variables, such as [GeoIPOverrides](../parameters.mdx#GeoIPOverrides) and [Origin.Exports](../parameters.mdx##Origin-Exports). For these parameters, use a configuration file instead.
+
+### Configure via the configuration file
+
+Pelican servers can also be configured via a configuration file in `yaml`. The default location that Pelican looks for a configuration file is `/etc/pelican/pelican.yaml` if Pelican runs as the root user (which is the case when the Pelican server is running in a container). If the Pelican server runs as a non-root user, the default location is `~/.config/pelican/pelican.yaml`.
+
+It is recommended that admins bind a Pelican configuration file on the host machine to the container running the Pelican server. Follow the steps below for instructions.
+
+1. Prepare an `yaml` file on the host machine:
```
Comment (turetske) [inline docs/pages/install/docker.mdx:119]:
```suggestion
1. Prepare a `yaml` file on the host machine:
```

**Review by turetske — CHANGES_REQUESTED** [review]

**Review by jhiemstrawisc — APPROVED** [review]

**Comment by haoming29** [comment]
@turetske could you take another look? GitHub requires your approval to merge this

**Review by turetske — APPROVED** [review]

**Pull request merged**